packages/flutter_driver/lib/src/driver/vmservice_driver.dart

```diff
@@ -306,20 +303,20 @@ class VMServiceFlutterDriver extends FlutterDriver {
 
   @override
   Future<Map<String, dynamic>> sendCommand(Command command) async {
-    Map<String, dynamic> response;
+    late Map<String, dynamic> response;
     try {
       final Map<String, String> serialized = command.serialize();
       _logCommunication('>>> $serialized');
       final Future<Map<String, dynamic>> future = _serviceClient.callServiceExtension(
         _flutterExtensionMethodName,
         isolateId: _appIsolate.id,
         args: serialized,
-      ).then<Map<String, dynamic>>((vms.Response value) => value.json);
-      response = await _warnIfSlow<Map<String, dynamic>>(
+      ).then<Map<String, dynamic>>((vms.Response value) => value.json!);
```

Comment:
We are seeing stack traces in our build failures that come from the `_warnIfSlow` method following this when it evaluates the future. The error is that a `Map<String, dynamic>` can't be null, so I believe that `!` operator on this line may be encountering a null value here and failing the NNBD assertion.